resource-watch/resource-watch
# Favorites and Collections requests optimization (called too many times)

## Overview
This PR fixes the issues when favorites and collection requests call many times.
## Testing instructions
Login to the system.
Go to http://localhost:9000/data/explore
Add datasets to favorites or collections.  
Go to http://localhost:9000/myrw/
Check all requests when you change tabs. Favourites and collections will not in the list. 
Go to http://localhost:9000/myrw/collections. Check all actions here (fetch, add, edit, delete). Collections should update only on http://localhost:9000/myrw/collections page.

## [Pivotal task](https://www.pivotaltracker.com/n/projects/1374154/stories/171126543)

---

## Checklist before submitting
